[2024-01-26T16:15:04.192-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-26T19:00:00+00:00/task_id=invoke_lambda_function permission to 509
[2024-01-26T16:15:04.238-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-26T19:00:00+00:00/task_id=invoke_lambda_function permission to 509
[2024-01-26T16:15:04.256-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: how-desafio-03.invoke_lambda_function scheduled__2024-01-26T19:00:00+00:00 [queued]>
[2024-01-26T16:15:04.267-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: how-desafio-03.invoke_lambda_function scheduled__2024-01-26T19:00:00+00:00 [queued]>
[2024-01-26T16:15:04.268-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-26T16:15:04.295-0300] {taskinstance.py:1382} INFO - Executing <Task(LambdaInvokeFunctionOperator): invoke_lambda_function> on 2024-01-26 19:00:00+00:00
[2024-01-26T16:15:04.297-0300] {standard_task_runner.py:57} INFO - Started process 864 to run task
[2024-01-26T16:15:04.299-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'how-desafio-03', 'invoke_lambda_function', 'scheduled__2024-01-26T19:00:00+00:00', '--job-id', '259', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpgbsi_o8l']
[2024-01-26T16:15:04.300-0300] {standard_task_runner.py:85} INFO - Job 259: Subtask invoke_lambda_function
[2024-01-26T16:15:04.364-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-26T19:00:00+00:00/task_id=invoke_lambda_function permission to 509
[2024-01-26T16:15:04.365-0300] {task_command.py:416} INFO - Running <TaskInstance: how-desafio-03.invoke_lambda_function scheduled__2024-01-26T19:00:00+00:00 [running]> on host 14111-NB.
[2024-01-26T16:15:04.463-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='how-desafio-03' AIRFLOW_CTX_TASK_ID='invoke_lambda_function' AIRFLOW_CTX_EXECUTION_DATE='2024-01-26T19:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-26T19:00:00+00:00'
[2024-01-26T16:15:04.464-0300] {lambda_function.py:211} INFO - Invoking AWS Lambda function: how-desafio-03 with payload: {"date": "1706296500"}
[2024-01-26T16:15:04.472-0300] {base.py:73} INFO - Using connection ID 'aws_default' for task execution.
[2024-01-26T16:15:04.473-0300] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-01-26T16:15:05.521-0300] {lambda_function.py:220} INFO - Lambda response metadata: {'RequestId': '8f7fa2d4-853b-4fe9-97f8-83cf7ae2b841', 'HTTPStatusCode': 202, 'HTTPHeaders': {'date': 'Fri, 26 Jan 2024 19:15:29 GMT', 'content-length': '0', 'connection': 'keep-alive', 'x-amzn-requestid': '8f7fa2d4-853b-4fe9-97f8-83cf7ae2b841', 'x-amzn-remapped-content-length': '0', 'x-amzn-trace-id': 'root=1-65b404d1-1c6fe65904bc54ea0d5fbc4b;sampled=0'}, 'RetryAttempts': 0}
[2024-01-26T16:15:05.522-0300] {lambda_function.py:230} INFO - Lambda function invocation succeeded: {'RequestId': '8f7fa2d4-853b-4fe9-97f8-83cf7ae2b841', 'HTTPStatusCode': 202, 'HTTPHeaders': {'date': 'Fri, 26 Jan 2024 19:15:29 GMT', 'content-length': '0', 'connection': 'keep-alive', 'x-amzn-requestid': '8f7fa2d4-853b-4fe9-97f8-83cf7ae2b841', 'x-amzn-remapped-content-length': '0', 'x-amzn-trace-id': 'root=1-65b404d1-1c6fe65904bc54ea0d5fbc4b;sampled=0'}, 'RetryAttempts': 0}
[2024-01-26T16:15:05.569-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=how-desafio-03, task_id=invoke_lambda_function, execution_date=20240126T190000, start_date=20240126T191504, end_date=20240126T191505
[2024-01-26T16:15:05.600-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-26T16:15:05.627-0300] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
